default/python/AI/AIDependencies.py

```diff
@@ -77,21 +77,21 @@
 # boni that scale with planet size
 POP_SIZE_MOD_MAP = {
     # "Effect": [uninhab, hostile, poor, adequate, good]
-    "environment_bonus": [0,-4,-2, 0, 3],
-    "GRO_SUBTER_HAB":    [0, 1, 1, 1, 1],
+    "environment_bonus": [0, -4, -2, 0, 3],
+    "GRO_SUBTER_HAB": [0, 1, 1, 1, 1],
     "GRO_SYMBIOTIC_BIO": [0, 0, 1, 1, 1],
     "GRO_XENO_GENETICS": [0, 1, 2, 2, 0],
-    "GRO_XENO_HYBRID":   [0, 2, 1, 0, 0],
-    "GRO_CYBORG":        [0, 2, 0, 0, 0],
-    "CON_NDIM_STRUC":    [0, 2, 2, 2, 2],
-    "CON_ORBITAL_HAB":   [0, 1, 1, 1, 1],
+    "GRO_XENO_HYBRID": [0, 2, 1, 0, 0],
+    "GRO_CYBORG": [0, 2, 0, 0, 0],
+    "CON_NDIM_STRUC": [0, 2, 2, 2, 2],
+    "CON_ORBITAL_HAB": [0, 1, 1, 1, 1],
```

Comment:
I agree with this: dependencies needs to be completely accessible for non coders, it's set out as a readable table to let scripters like me make amendments easily.